apps/shared/style/fonts.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
+  font-family: "Readex Pro";
+  src: url(/fonts/Readex_Pro/ReadexPro-VariableFont_wght.ttf);
+}
+
+* {
+  --sl-font-sans: "Readex Pro", sans-serif;
+}
+
+html, body {
+  font-family: var(--sl-font-sans);
+}
```

Comment:
Since you have a rule here for html and body, all the ones adding the same font for the same tags in style.css are redundant, right? If so they should be removed instead of updated.

Reply:
yes